### ISSUE TITLE
With the latest version in use, an indication for a new version is displayed

### ISSUE THREAD

**Issue by CruxH** [amir20/dozzle#2681]
**Describe the bug**
With the latest version in use, an indication for a new version is displayed:

![image](https://github.com/amir20/dozzle/assets/7459949/04844729-8e1a-452d-857c-98d01bdc7aa0)

**Comment by amir20** [comment]
I bet I know what happened. I cache the release for an hour. You must have gotten the releases, then updated Dozzle, and gone to the settings page. Right?

If you go to it now, it should be ok. 

Small bug, but yes should be fixed.

**Comment by CruxH** [comment]
Yes, I had the Dozzle tab open, updated to 6.0.6, and then refreshed the tab. Currently, the indicator for the new version is no longer visible.

**Comment by amir20** [comment]
Releasing a patch which should fix most of the bugs.

**Comment by CruxH** [comment]
Sorry, it appears that this issue has not been resolved. The situation remains the same as previously described: with Dozzle open in a background tab, updating, and refreshing the Dozzle tab.

![image](https://github.com/amir20/dozzle/assets/7459949/ff0f9f20-6b61-4679-8315-e084627b4980)

**Comment by amir20** [comment]
It will remain for another hour because the caching in `6.0.6` was done via HTTP so the browser was caching. I have moved that cache to API now. But the HTTP cache will still be active for another hour.
